include/swift/AST/DiagnosticsSema.def

```diff
@@ -4737,8 +4737,8 @@ ERROR(any_not_existential,none,
        "'any' has no effect on %select{concrete type|type parameter}0 %1",
        (bool, Type))
 ERROR(existential_requires_any,none,
-      "%select{protocol |}1%0 as a type must be explicitly marked as 'any'",
-      (Type, bool))
+      "use of %select{protocol |}2%0 as a type must be written %1",
```

Comment:
I'm not sure about the exact phrasing of this, but I think it's an improvement. Happy for suggestions of how to make it even better.

Reply:
I agree "must be written" is an improvement! I've also been wanting to change the phrase "protocol X as a type" to something different because I want people to stop thinking about protocols as types, but I also don't want the term "existential type" to show up in diagnostics...